xds/src/main/java/io/grpc/xds/XdsClientMetricReporterImpl.java

```diff
@@ -203,7 +200,6 @@ static final class MetricReporterCallback implements ResourceCallback,
     }
 
     // TODO(dnvindhya): include the "authority" label once xds.authority is available.
-    @Override
     public void reportResourceCountGauge(long resourceCount, String cacheState,
```

Comment:
private or package-private? Or just inline it? It doesn't get any benefit from being here instead of the parent class.

Reply:
Updated to package-private.